pkg/collector/node.go

```diff
@@ -1,11 +1,30 @@
 package collector
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/node_exporter/collector"
 )
 
+var whitelist = map[string]bool{
+	"node_network_receive_bytes_total":  true,
```

Comment:
Seems odd to use a map for a whitelist. if you are going to use a map with a bool value, why not actually check the value of true or false when evaluating the whitelist below?

Reply:
:point_up:

Reply:
The map is only being used as a Set. The value is useless.

Reply:
I get that, but you are never doing a lookup so it doesn't really make sense to use a map vs just an array or struct imo

Reply:
:point_up:

Reply:
Yep. I realize that I changed it from a lookup to a loop now. Pushed a change.